hooks/nuke/publish_files.py

```diff
@@ -198,11 +198,9 @@ def finalize(self, task_settings, item):
 
         super(NukePublishFilesPlugin, self).finalize(task_settings, item)
 
-        # only version up the file if the publish went through successfully.
-        if item.properties.get("sg_publish_data"):
-            # insert the path into the properties
-            if item.type == 'file.nuke':
-                item.properties["next_version_path"] = self._bump_file_version(path)
+        # insert the path into the properties
+        if item.type == 'file.nuke':
+            item.properties["next_version_path"] = self._bump_file_version(path)
```

Comment:
Since we are re-raising any exceptions in publish(), finalise() will never be called unless publish succeeds, i.e. item has 'sg_publish_data'